spyder/utils/sourcecode.py

```diff
@@ -9,7 +9,10 @@
 """
 
 import re
-
+import os
+from spyder.py3compat import PY2

```

Comment:
Please add a blank line before this one, to separate standard library imports from Spyder ones.